dff/context_storages/context_schema.py

```diff
@@ -0,0 +1,270 @@
+from asyncio import gather
+from datetime import datetime
+from uuid import uuid4
+from enum import Enum
+from pydantic import BaseModel, Field
+from typing import Any, Coroutine, List, Dict, Optional, Callable, Tuple, Union, Awaitable
+from typing_extensions import Literal
+
+from dff.script import Context
+
+ALL_ITEMS = "__all__"
+"""
+`__all__` - the default value for all `DictSchemaField`s:
+it means that all keys of the dictionary or list will be read or written.
+Can be used as a value of `subscript` parameter for `DictSchemaField`s and `ListSchemaField`s.
+"""
+
+_ReadPackedContextFunction = Callable[[str], Awaitable[Tuple[Dict, Optional[str]]]]
```

Comment:
It would make sense to additionally document the difference between the `context` table and the `logs` table in the module docstring, since it might be unclear for a regular user, but understanding it is important for module usage